internal/runner/runner_race_test.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/vdaas/vald/internal/test/goleak"
 )
 
-func TestDo(t *testing.T) {
+func TestDo_for_race(t *testing.T) {
```

Comment:
I have fixed function name because of CI fails.
The reason why CI fails was because there was a same function in to the same package.

https://github.com/vdaas/vald/blob/master/internal/runner/runner_test.go#L284